Peanut update error

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

Peanut

### 📂 What was the exact command used to execute the script?

update

### ⚙️ What settings are you using?

- [x] Default Settings
- [ ] Advanced Settings

### 🖥️ Which Linux distribution are you using?

_No response_

### 📝 Provide a clear and concise description of the issue.

update script of peanut is failing

### 🔄 Steps to reproduce the issue.

run update from the shell
updating to 5.7.2 fails




### ❌ Paste the full error output (if available).

⠸ Updating PeaNUT to v5.7.2tar: option '--strip-components' requires an argument
Try 'tar --help' or 'tar --usage' for more information.
⠼ Updating PeaNUT to v5.7.2
[ERROR] in line 35: exit code 0: while executing command tar -xzf peanut.tar.gz -C /opt/peanut --strip-components

### 🖼️ Additional context (optional).

solution is to change the line 35 to 

--strip-components=1

--- a/ct/peanut.sh
+++ b/ct/peanut.sh
@@ -32,7 +32,7 @@ function update_script() {
     msg_info "Updating $APP to ${RELEASE}"
     systemctl stop peanut
     curl -fsSL "https://api.github.com/repos/Brandawg93/PeaNUT/tarball/${RELEASE}" -o "peanut.tar.gz"
-    tar -xzf peanut.tar.gz -C /opt/peanut --strip-components
+    tar -xzf peanut.tar.gz -C /opt/peanut --strip-components=1
     rm peanut.tar.gz
     cd /opt/peanut
     $STD pnpm i

--- a/install/peanut-install.sh
+++ b/install/peanut-install.sh
@@ -34,7 +34,7 @@ msg_info "Installing Peanut"
 RELEASE=$(curl -fsSL https://api.github.com/repos/Brandawg93/PeaNUT/releases/latest | grep '"tag_name":' | cut -d'"' -f4)
 curl -fsSL "https://api.github.com/repos/Brandawg93/PeaNUT/tarball/${RELEASE}" -o "peanut.tar.gz"
 mkdir -p /opt/peanut
-tar -xzf peanut.tar.gz -C /opt/peanut --strip-components 1
+tar -xzf peanut.tar.gz -C /opt/peanut --strip-components=1
 rm peanut.tar.gz
 cd /opt/peanut
 $STD npm install -g pnpm

